docs/components/best-practices/development/dealing-with-problems-and-exceptions.md

```diff
@@ -248,10 +248,12 @@ The Saga pattern describes long-running transactions in distributed systems. The
 
 Camunda supports this through BPMN compensation events, which can link tasks with their undo tasks.
 
-:::caution Camunda 7 Only
+:::caution Camunda 7 only
 Compensation is [not yet supported in Camunda 8](/components/modeler/bpmn/bpmn-coverage.md) and only available in Camunda 7.
 :::
 
+<!--- Do we feel this warrants its own page as well? As this is related tot he rest of the document and is quite small, I would opt to leave it alone. --->
```

Comment:
Support for compensation events should be coming in 8.5 - https://github.com/camunda/product-hub/issues/219. This can actually be removed soon! 

Remove it in a separate PR, add the hold + target label, and link it back to the epic.